### ISSUE TITLE
Document pull request workflow

### ISSUE THREAD

**Issue by jtniehof** [spacepy/spacepy#40]
We've been running this a little informally (which is a lot more formal than before!) but we should probably document how we're handling PRs. Stuff like CI build should work, merge should be done by somebody other than the person submitting the PR, and I suggest we leave PRs open for 24 hours minimum so people can comment.

I don't know if we want to start this in a wiki or stick to Sphinx.

**Comment by balarsen** [comment]
Not a huge preference but I prefer Sphinx just so all the docs live together.

**Comment by drsteve** [comment]
We should also make a PR template (probably also an issue template). They're pretty straightforward and would live in a ```.github``` directory. Check out the templates on other (big) projects and see what you think.

**Comment by balarsen** [comment]
Isn't this done? @drsteve

**Comment by drsteve** [comment]
Nope. We have the PR template, but we haven't got documentation that says,
for example, that all PRs need to be reviewed and that the requestor can't
merge their own PR.
